### PR TITLE
Add geographiclib python dependency

### PR DESCRIPTION
**Pull request by anassinator**

### CODE REVIEW

**Review by wjwwood — COMMENTED** [review]
Can you please add rules for Debian:

- https://packages.debian.org/stretch/python-geographiclib
- https://packages.debian.org/jessie/python-geographiclib

Also, Fedora if you can find an alternative.

File: rosdep/python.yaml [hunk 1]
```diff
@@ -912,6 +912,21 @@ python-gdown-pip:
   ubuntu:
     pip:
       packages: [gdown]
+python-geographiclib:
+  ubuntu: [python-geographiclib]
+python-geographiclib-pip:
+  debian:
+    pip:
+      packages: [geographiclib]
+  fedora:
```
Comment (tfoote) [inline rosdep/python.yaml]: The fedora package is here: https://apps.fedoraproject.org/packages/python-GeographicLib and appears to be available for all recent platforms

**Review by tfoote — CHANGES_REQUESTED** [review]

**Comment by anassinator** [comment]
Done, but not sure if it's right?

**Review by wjwwood — APPROVED** [review]
lgtm, thanks!

**Comment by tfoote** [comment]
The new rules for fedora and debian look good. Could you remove the `-pip` rule. If the package exists in upstream packaging the pip install will mask it and can side effect other installations.

**Comment by anassinator** [comment]
What about OSX?

**Comment by anassinator** [comment]
I've removed it as requested regardless.

**Review by tfoote — APPROVED** [review]

**Comment by tfoote** [comment]
For OSX if someone needs/wants it we usually look for a homebrew package or pip if necessary and someone needs it on the main rule. The `-pip` separate rule is for things that don't have mainline packages at the moment but we think they might in the future. Using the `-pip` extension make sure they don't collide in the future.

**Pull request merged**